ecommerce/serializers.py

```diff
@@ -694,3 +729,87 @@ def get_consent_text(self, instance):
     class Meta:
         fields = ["consent_date", "id", "company", "consent_text"]
         model = models.DataConsentUser
+
+
+class LineSummarySerializer(serializers.ModelSerializer):
+    """ Summary serializer for Line model """
+
+    product_version = ProductVersionSummarySerializer()
+
+    class Meta:
+        model = models.Line
+        fields = ["product_version", "quantity"]
+
+
+class OrderReceiptSerializer(serializers.ModelSerializer):
+    """ Serializer for extracting receipt info from an Order object"""
+
+    lines = serializers.SerializerMethodField()
+    purchaser = serializers.SerializerMethodField()
+    coupon = serializers.SerializerMethodField()
+    order = serializers.SerializerMethodField()
+    receipt = serializers.SerializerMethodField()
+
+    def get_receipt(self, instance):
+        """Get receipt information if it exists"""
+        receipt = instance.receipt_set.order_by("-created_on").first()
```

Comment:
When will we have multiple receipts for one order? I'm confused here. Is it for Program only?

Reply:
We shouldn't, ideally, although the code does not restrict you from creating multiple receipts for an order. This is just to safeguard from an edge case where multiple receipts exist for an order.